ui/DesignSystem/Primitive/Input/Text.svelte

```diff
@@ -71,12 +72,13 @@
     {disabled}
     on:change
     on:input />
-  {#if variant === 'handle' && value && avatar}
+  {#if variant === 'handle'}
```

Comment:
Instead of all these checks in here for a variant, wouldn't it be cleaner to refactor it into its own component?

Reply:
I'd prefer to keep them in the same component to have consistent styling.